### PR TITLE
reformatted file to pass flake8 (command in body)

### PR DESCRIPTION
**Pull request by Jlrine2**
flake8 --max-line-length=120 --import-order-style=pycharm --statistics

### CODE REVIEW

File: ArcGIS-toolbox/ASF_Tools.pyt [hunk 1]
```diff
@@ -778,7 +797,6 @@ class RGBDecomp(object):
         arcpy.env.workspace = indir
         indirbase = os.path.basename(indir)
         # Create a scratch directory for intermediate files
-        scratch = arcpy.CreateFolder_management(indir, "temp")
```
Comment (asjohnston-asf) [inline ArcGIS-toolbox/ASF_Tools.pyt:781]: `CreateFolder_management` might have side effects that are needed even if the returned `scratch` value isn't used

File: ArcGIS-toolbox/ASF_Tools.pyt [hunk 2]
```diff
@@ -415,21 +431,16 @@ class ReclassifyRTC(object):
...
-        thresh = parameters[3].value
         outYN = parameters[4].valueAsText
 
-        threshstr = str(thresh)
-
         # Run the code to reclassify the image
-        
-        rcname = str(rc_outpath+'\\'+rc_outname)
-        values = "-1000.000000 %s 1;%s 1000.000000 NODATA" % (threshstr, threshstr)
-        reclass = arcpy.gp.Reclassify_sa(inRTC, "VALUE", values, rcname, "DATA")
```
Comment (asjohnston-asf) [inline ArcGIS-toolbox/ASF_Tools.pyt:432]: Per Heidi `Reclassify_sa` does needed work, even if the returned `reclass` value isn't used

Comment (Jlrine2) [reply ArcGIS-toolbox/ASF_Tools.pyt:432]: @hjkristenson Is it okay to just make the funciton call without assignment here?

File: ArcGIS-toolbox/ASF_Tools.pyt (hunk 1 above)
Comment (Jlrine2) [reply ArcGIS-toolbox/ASF_Tools.pyt:781]: @hjkristenson is it okay to call the function without assignment here

File: ArcGIS-toolbox/ASF_Tools.pyt [hunk 3]
```diff
@@ -735,7 +755,7 @@ class RGBDecomp(object):
         if not parameters[3].altered:
             parameters[3].value = -24
 
-        # Set the default output directory to be the same as the input directory
```
Comment (hjkristenson) [inline ArcGIS-toolbox/ASF_Tools.pyt]: Why was this changed to directoryzipfile? Should just be directory.

File: ArcGIS-toolbox/ASF_Tools.pyt [hunk 4]
```diff
@@ -762,12 +782,11 @@ class RGBDecomp(object):
         indir = parameters[0].valueAsText
         scale = parameters[1].valueAsText
         pol = parameters[2].valueAsText
-        rb_thresh_db = parameters[3].valueAsText
```
Comment (hjkristenson) [inline ArcGIS-toolbox/ASF_Tools.pyt:765]: This is definitely a required variable.
Ah, I see. I hard-coded in a value when I should have used this input variable on line 840. 
Instead of -2.4 it should be rb_thresh_db/10.

File: ArcGIS-toolbox/ASF_Tools.pyt (hunk 1 above)
Comment (hjkristenson) [inline ArcGIS-toolbox/ASF_Tools.pyt:781]: Does this automatically happen in the os.path.join function? If so, that's fine. But this directory needs to be created one way or another. 
Ah, I see, it was removed because it was set to be a variable, but I changed the scratchpath to not use this variable. If scratchpath doesn't generate a directory, this line should be added back in, but doesn't need to be a variable.

File: ArcGIS-toolbox/ASF_Tools.pyt [hunk 5]
```diff
@@ -87,98 +93,100 @@ class UnzipFiles(object):
...
         params = [p_inpath, p_inscale, p_outscale, p_outdir, p_outname, outYN, outlayer]
         return params
 
     def rasConvert(self):
         """Conversion function"""
-        
+
         if self.inscale == 'Power':
             if self.outscale == 'Amplitude':
                 outSqRt = arcpy.sa.SquareRoot(self.inpath)
```
Comment (hjkristenson) [inline ArcGIS-toolbox/ASF_Tools.pyt:192]: This section is one place where I used pseudo-camelCase. These variable names could be changed if desired.

**Review by hjkristenson — CHANGES_REQUESTED** [review]
Many of the outstanding issues have to do with "unused" variables, and have been addressed individually in the comments.

File: ArcGIS-toolbox/ASF_Tools.pyt [hunk 6]
```diff
@@ -786,6 +786,7 @@ class RGBDecomp(object):
         indir = parameters[0].valueAsText
         scale = parameters[1].valueAsText
         pol = parameters[2].valueAsText
+        rb_thresh_db = parameters[3].valueAsText
```
Comment (hjkristenson) [inline ArcGIS-toolbox/ASF_Tools.pyt]: Change this line to remove the "AsText" part.
rb_thresh_db = parameters[3].value

File: ArcGIS-toolbox/ASF_Tools.pyt [hunk 7]
```diff
@@ -861,7 +861,7 @@ class RGBDecomp(object):
         arcpy.AddMessage("Pixel cleanup complete. Generating spatial masks...")
 
         # Generate spatial masks based on red/blue threshold
-        rb_thresh = math.pow(10, rb_thresh_db / 10)
+        rb_thresh = math.pow(10, float(rb_thresh_db) / 10)
```
Comment (hjkristenson) [inline ArcGIS-toolbox/ASF_Tools.pyt:864]: It would be more robust to change the parameter format to GPDouble from GPLong (line 670). I would also change the displayName to read "...(default: -24.0 dB)" 
Also change line 760 to be -24.0 instead of -24, for clarity.
This solves the integer division problem, and also allows users finer control of the threshold parameter by supporting decimals.

**Review by hjkristenson — APPROVED** [review]

**Review by asjohnston-asf — APPROVED** [review]
